autodepgraph/node.py

```diff
@@ -31,27 +31,36 @@ def __init__(self, name, verbose=False):
                            initial_value=[],
                            vals=vals.Lists(vals.Strings()),
                            parameter_class=ManualParameter)
+        self._parenth_graph = None
 
     def __call__(self, verbose=False):
         return self.execute_node(verbose=verbose)
 
+    def update_monitor(self):
```

Comment:
Why does the node have an update monitor method? I think only the graph instance should be able to do that.